src/templates/profile.ts

```diff
@@ -15,3 +16,53 @@ ${name} usecase
 usecase ${name} {}
 `;
 }
+
+export function complete(name: string): string {
+  return `
+"""
+${name} usecase
+"""
+usecase ${name} {
+  input {
+    "field title"
+    foo! string 
+
+    bar 
```

Comment:
Unsure about this one... I don't think I've seen the `field` directive in use yet :sweat_smile: So I'd probably leave it out of a "beginner's template"?